app/src/main/java/com/pajato/android/gamechat/chat/model/Room.java

```diff
@@ -106,7 +106,13 @@ public Room(final String key, final String owner, final String name, final Strin
             case ME:
                 return account.getDisplayName();
             case PRIVATE:
-                return memberNames(account);
+                // if name is not set, try to get memberNames value
```

Comment:
fwiw, I'd be inclined to combine the two comments, lose the curly braces around the first if clause.